src/main/java/com/uid2/admin/secret/SaltRotation.java

```diff
@@ -94,86 +125,142 @@ private String calculatePreviousSalt(SaltEntry salt, boolean shouldRotate, long
             return salt.currentSalt();
         }
         long age = nextEffective - salt.lastUpdated();
```

Comment:
Should we add a SaltEntry.ageInDays(asOf) method?

Reply:
SaltEntry is in Shared and currently this isn't needed anywhere except Admin. I guess we might need this for Operator though so might be ok.

Reply:
I think Operator checks salt age as well, so would be broadly useful

Reply:
We could just have one Admin helper function for this if we don't want to add it to shared